pkg/registries/registries.go

```diff
@@ -124,9 +143,17 @@ func EditRegistriesConfig(config *sysregistriesv2.V2RegistriesConf, insecureRegi
 		return err
 	}
 	for _, mirrorSet := range mirrorSets {
+		// If source is a wildcard entry skip it as we don't currently support wildcards for mirror configuration
+		if strings.Contains(mirrorSet.Source, "*") {
+			continue
+		}
 		reg := getRegistryEntry(mirrorSet.Source)
 		reg.MirrorByDigestOnly = true
 		for _, mirror := range mirrorSet.Mirrors {
+			// If a mirror is a wildcard entry skip it as we don't currently support wildcards for mirror configuration
+			if strings.Contains(mirror, "*") {
```

Comment:
Why is it OK to _silently_ ignore user input?

Reply:
Fixed.
No longer silently ignoring user input.